src/background/assessment-data-converter.ts

```diff
@@ -167,7 +167,7 @@ export class AssessmentDataConverter {
             isCapturedByUser: false,
             failureSummary: ruleResult.failureSummary,
             isVisualizationEnabled: false,
-            isVisible: elementAxeResult.isVisible,
+            isVisible: true,
```

Comment:
I decided to leave the "isVisible" property in the test step result interface as a matter of not increasing the size of this PR even further; previously this would always be true anyways, so this made sense to me.